logistics_planning_base/views/logistics_schedule_views.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="logistics_schedule_view_tree" model="ir.ui.view">
+        <field name="name">logistics.schedule.tree</field>
+        <field name="model">logistics.schedule</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="picking_id" />
+                <field name="partner_id" />
```

Comment:
Add for both tree and form view `stock_move_id`, with a restricted domain by partner, product, not linked to any logistics schedule yet and in state='done'.